app/src/main/java/it/quip/android/partials/EndlessScrollHandler.java

```diff
@@ -0,0 +1,47 @@
+package it.quip.android.partials;
```

Comment:
And it should be `EndlessScrollListener` since it inherits from the base listener